receiver/prometheusexec/README.md

```diff
@@ -0,0 +1,126 @@
+# This receiver is currently under development, do not use.
+// TODO: Remove above message when done
+
+# Prometheus_exec Receiver
+
+This receiver makes it easy for a user to collect metrics with the Collector from third-party services such as MySQL, Apache, Redis, etc. It's meant for people who want a plug-and-play solution to getting metrics from those third-party services that don't natively export metrics or speak any instrumentation protocols. Through the configuration file, you can indicate which binaries to run (usually [Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/), which are custom binaries that will expose the services' metrics to the Prometheus protocol). It supports starting binaries with flags and environment variables, retrying them with exponentional backoff if they crash, string templating, and random port assignments. If you do not need to spawn the binaries locally, please condider using the [core Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/prometheusreceiver) or the [Simple Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver/simpleprometheusreceiver).
```

Comment:
I have a little bit of trouble reconciling that opening sentence. As someone reading this README for this first time, I would get very confused by this, as, for example, there is already a Redis receiver in this same repo, i.e. we are talking about "Redis -> Prom Exporter -> Prom Receiver -> Export" vs "Redis -> Redis Receiver -> Export"

I think it would be clearer to open with something like "collecting metrics **via third party applications that expose prometheus metrics**" (or maybe just "collecting metrics via prometheus exporter plugins, or similar")?

And due to the multiple different ways you can generate metrics from applications, it may be worth adding a line that justifies why this is useful - e.g. "to make use of the entire existing ecosystem of Prometheus exporter plugins" or something like that.

Reply:
Thanks, I fixed the wording for multiple sentences in this paragraph!